charts/zot/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: v2.0.0
 description: A Helm chart for Kubernetes
 name: zot
 type: application
-version: 0.1.42
+version: 0.1.43
```

Comment:
@Kortekaasy can you pls rebase and bump this to 0.1.44?

Reply:
Otherwise, lgtm

Reply:
I've rebased on the upstream branch and bumped the version to 0.1.44